src/App.js

```diff
@@ -4,9 +4,39 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    movies: moviesFromServer,
+    query: '',
+  };
 
   render() {
+    let { movies } = this.state;
+    const { query } = this.state;
```

Comment:
```suggestion
    const { movies, query } = this.state;
```
эти переменные не будут меняться